yarn.lock

```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
We can delete this `yarn.lock` file - I'm guessing you ran a `yarn` command from the root directory by accident at some point :D